src/DifferenceFormatter.php

```diff
@@ -64,17 +64,17 @@ public function diffForHumans(
                 $unit = 'year';
                 $count = $diffInterval->y;
                 break;
-            case $diffInterval->m > 0:
+            case $diffInterval->days >= ChronosInterface::DAYS_PER_WEEK * 8:
```

Comment:
In #307 you mentioned moving to months after 4 weeks not 8. What led to changing your mind?

Reply:
The table had 4 weeks but my recommendation was to give weeks a little bit more space:

> So here's my proposal: Increase days to be 1..21, give weeks a slight bump (like 3..6 or 3..8) and leave the rest alone.

I think at this point it's very subjective. If we want to stick to the same precision then we would go up to "60 weeks", which isn't very human :) If you prefer 4 weeks I'm glad to switch it; I just went with what sounds colloquial.

Reply:
@markstory checking back in here

Reply:
Hmm, '60 weeks' is pretty long. To me (in Canada) 8 weeks is probably the right duration. I could see arguments for going to 3 months :shrug: